index.html

```diff
@@ -0,0 +1,187 @@
+</head>
+<body>
+    <div class="header-section1-container">
+        <!-- header -->
+        <header class="header">
+            <div class="humburger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
+
+            <nav class="mobile-nav">
+<ul class="ul">
+    <li class="li"><a href="about.html" class="a">About</a></li>
+    <li class="li"><a href="#program.html" class="a">Program</a></li>
+    <li class="li"><a href="sponsor.html" class="a">Sponsor</a></li>
+    <li class="li"><a href="" class="a">New</a></li>
+    <li class="li"><a href="" class="a">Join</a></li>
+    <li class="li li-cc"><a class="a a-cc" href="">TMS Campaign</a></li>
+</ul>
+<div class="closeBtn">X</div>
+            </nav>
+
+            <nav class="desktop-nav">
+                <ul class="black-row">
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+
+                <div class="white-row">
+                    <a href="index.html">
+                        <img src="images/tmsLogo.png" alt="logo" class="logo">
+                    </a>
+                    <ul class="list">
+                        <li class="li">
+                          <a class="a" href="about.html">About</a>
+                        </li>
+                        <li class="li"><a href="#program.html" class="a">Program</a></li>
+                        <li class="li"><a href="sponsor.html" class="a">Sponsor</a></li>
+                        <li class="li"><a href="" class="a">New</a></li>
+                        <li class="li"><a href="" class="a">Join</a></li>
+                        <li class="li li-cc"><a class="a a-cc" href="">TMS Campaign</a></li>
+                      </ul>
+                </div>
+            </nav>
+        </header>
+        <section class="section-1">
+            <div class="hello">"Hello! Sharing world"</div>
+            <h1 class="h1">TechMingle Summit 2023</h1>
+            <p class="p">
+         The event aims to foster a collaborative and interactive environment where attendees can network, learn, and share knowledge about the latest trends, tools, and practices in web development.
+         Creating a positive change with people from over 80 countries is
+         taking place in October, in Rwanda.
+            </p>
+            <div class="date">2023.10.15(THU) ~ 16(FRI)</div>
+            <div class="location">
+              @ Kigali Convention Center
+            </div>
+          </section>
+    </div>
+    
+    <!-- Main program section -->
+    <section class="section-2" id="program">
+        <div class="h2-red-line">
+            <h2 class="h2">Main Program</h2>
+            <hr class="red-line">
+        </div>
+
+        <div class="container">
```

Comment:
- [ ] Kindly modify the height on the containers to match with the behance design

Your design

![image](https://github.com/gilberthappi/TechMingle_Summit/assets/93630700/d413a573-4631-4e20-9971-0c2e5052cf29)

Behance design

![image](https://github.com/gilberthappi/TechMingle_Summit/assets/93630700/af0c8759-636b-4b9e-9915-bf5ee9f469f2)